src/main/java/io/candydoc/ddd/bounded_context/BoundedContextExtractor.java

```diff
@@ -24,16 +25,18 @@ public List<Event> extract(ExtractDDDConcepts command) {
         .collect(Collectors.toUnmodifiableList());
   }
 
+  public List<Event> extract(ExtractBoundedContexts command) {
+    return extractBoundedContexts(command.getPackageToScan()).stream()
+        .collect(Collectors.toUnmodifiableList());
+  }
+
   public List<Event> extractBoundedContexts(String packageToScan) {
     if (packageToScan.isBlank()) {
```

Comment:
Is it still necessary to have this condition + exception ? It is already in the UseCase

Reply:
This condition is still necessary in case of empty string in entry like packagesToScan="", I will change the exception message and a test for it so